nautobot/core/views/mixins.py

```diff
@@ -105,7 +103,7 @@ def get_required_permission(self):
         """
         queryset = self.get_queryset()
         try:
-            permissions = [PERMISSIONS_ACTION_MAP[self.action]]
+            permissions = [self.get_action()]
```

Comment:
Is it possible apps were catching the `ValueError` here and this change will break apps?